src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -202,18 +216,16 @@ public SnapshotBuilderConceptsResponse enumerateConcepts(
     return new SnapshotBuilderConceptsResponse().result(concepts);
   }
 
-  public int getRollupCountForCriteriaGroups(
-      UUID snapshotId,
-      List<List<SnapshotBuilderCriteriaGroup>> criteriaGroups,
-      AuthenticatedUserRequest userRequest) {
+  public int getRollupCountForCohorts(
+      UUID snapshotId, List<SnapshotBuilderCohort> cohorts, AuthenticatedUserRequest userRequest) {
     Snapshot snapshot = snapshotService.retrieve(snapshotId);
     SnapshotBuilderSettings snapshotBuilderSettings =
         snapshotBuilderSettingsDao.getBySnapshotId(snapshotId);
 
     Query query =
         queryBuilderFactory
             .criteriaQueryBuilder("person", snapshotBuilderSettings)
-            .generateRollupCountsQueryForCriteriaGroupsList(criteriaGroups);
+            .generateRollupCountsQueryForCohortList(cohorts);
```

Comment:
Similar to this method's name, I would use `ForCohorts` instead of `ForCohortList`